docs/mathematics/Kth-smallest.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: K-th smallest element in an array
+slug: /mathematics/Kth-smallest
+---
+
+## Introduction
+
+In this article we will be given a array and a number K we have to find the Kth smallest element in an array. 
```

Comment:
Add a description about what the Kth smallest element means?

Reply:
done